### PR TITLE
CFIC: Initial Article

### PR DESCRIPTION
**Pull request by fajar13k**
Start of Catch French Intermediate Cup on Group Stage state

---

### CODE REVIEW

File: wiki/Tournaments/CFC/CFIC_1/fr.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+| #17 | redjiii | ![][flag_FR] | ![][flag_FR] | floflim | **17:30** |
+| #22 | Sageru | ![][flag_FR] | ![][flag_FR] | Zard | **18:00** |
+
+-----------------------
+
+## Liste des maps
+
+### Phase de poules
+
+**[Télécharge le mappack ici! (55 MB)](https://mega.nz/#!FotnlQzR!bTxeqfGRbm-sOboVhq-AP31WP4r2ExNaGYNwpFrCIdU)**
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/fr.md]: "Téléchargez le mappack ici!" instead. The current sentence is at the first person, and makes it look like you're directly speaking to someone.

File: wiki/Tournaments/CFC/CFIC_1/fr.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  - **[Angus And Julia Stone - Big Jet Plane (Matt Neux Remix) (WildOne94, JBHyperion) \[Elevation\]](/beatmapsets/344833#fruits/761644)**
+
+------------------------------------------------------------------------
+
+## Règlement
+
+### Règles Générales
+
+1. Le tournoi prendra place dans un format de 1v1 double-élimination avec un seeding traditionnel.
+2. Le tournoi durera 7 weekends. Les matchs seront joués sous les paramètres **Score V2** et **Head to Head**.
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/fr.md]: "Le tournoi durera 5 weekends."

File: wiki/Tournaments/CFC/CFIC_1/en.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+| Designer | ![][flag_FR] [Lotchidych](/users/8375240) |
+| Statistician | ![][flag_FR] [FayeurS 5](/users/6177263) |
+| Referees | ![][flag_FR] [FayeurS 5](/users/6177263), ![][flag_FR] [Ryumi](/users/6596270), ![][flag_FR] [TLQ\_Yoshii](/users/7157133), ![][flag_CA] [Sinaeb](/users/1576095), ![][flag_FR] [Mimir](/users/7382734) |
+| Wiki | ![][flag_ID] [Naruk](/users/7100002) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/858507)
+- [CFC Discord server](https://discord.gg/3DewaX8)
+- [CFC Challonge brackets](https://osucfc.challonge.com/CFIC1)
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/en.md]: CFIC Challonge brackets

File: wiki/Tournaments/CFC/CFIC_1/en.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+| Streamers | ![][flag_FR] [FulgurentKille](/users/4096322), ![][flag_CA] [Sinaeb](/users/1576095) |
+| Designer | ![][flag_FR] [Lotchidych](/users/8375240) |
+| Statistician | ![][flag_FR] [FayeurS 5](/users/6177263) |
+| Referees | ![][flag_FR] [FayeurS 5](/users/6177263), ![][flag_FR] [Ryumi](/users/6596270), ![][flag_FR] [TLQ\_Yoshii](/users/7157133), ![][flag_CA] [Sinaeb](/users/1576095), ![][flag_FR] [Mimir](/users/7382734) |
+| Wiki | ![][flag_ID] [Naruk](/users/7100002) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/858507)
+- [CFC Discord server](https://discord.gg/3DewaX8)
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/en.md]: CFIC Discord server

File: wiki/Tournaments/CFC/CFIC_1/fr.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+| Streamers | ![][flag_FR] [FulgurentKille](/users/4096322), ![][flag_CA] [Sinaeb](/users/1576095) |
+| Designer | ![][flag_FR] [Lotchidych](/users/8375240) |
+| Statisticien | ![][flag_FR] [FayeurS 5](/users/6177263) |
+| Arbitres | ![][flag_FR] [FayeurS 5](/users/6177263), ![][flag_FR] [Ryumi](/users/6596270), ![][flag_FR] [TLQ\_Yoshii](/users/7157133), ![][flag_CA] [Sinaeb](/users/1576095), ![][flag_FR] [Mimir](/users/7382734) |
+| Wiki | ![][flag_ID] [Naruk](/users/7100002) |
+
+## Liens
+
+- [Fil de discussion](/community/forums/topics/858507)
+- [Discord du CFC](https://discord.gg/3DewaX8)
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/fr.md]: Discord du CFIC

File: wiki/Tournaments/CFC/CFIC_1/fr.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+- [Challonge du CFC](https://osucfc.challonge.com/CFIC1)
```
Comment (Kasumiikun) [inline wiki/Tournaments/CFC/CFIC_1/fr.md]: Challonge du CFIC

File: wiki/Tournaments/CFC/CFIC_1/en.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+The **Catch French Intermediate Cup** (***CFIC***) is an intermediate one-on-one French osu!catch tournament hosted by ![][flag_FR] [FayeurS 5](/users/6177263). It is the first installment and a new branch of the Catch French Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-01-20/2019-02-17 |
+| Live Drawings | 2019-02-23 (19:00 UTC+1) |
+| Group Stage | 2019-03-02 |
```
Comment (WalterToro) [inline wiki/Tournaments/CFC/CFIC_1/en.md]: Whoa, i missed this, this is from March 1st to 3rd

**Pull request merged**